packages/viewer/src/components.d.ts

```diff
@@ -2005,6 +2009,10 @@ declare namespace LocalJSX {
      * Emits an event when a provided oauth2 token is about to expire, or is about to expire, causing issues with establishing a websocket connection, or performing API calls.
      */
     onTokenExpired?: (event: VertexViewerCustomEvent<void>) => void;
+    /**
+     * Specifies the opacity, between 0 and 1, for a phantom feature, where 0 is completely hidden and 1 is completely visible.
+     */
+    phantomOpacity?: number;
```

Comment:
@jdm717 Curious on your thoughts here, if we should consider having this be a `phantomOptions` object, or if we should have the opacity right on here like this?

Reply:
I think I would prefer the `phantomOptions` route.

Reply:
Yeah, I think I'd agree with the `phantomOptions` route, since we'll likely add to those parameters down the road. That being said, the direction we're moving with these `options`-like properties does prevent a pure HTML based configuration of the viewer, which would be nice to maintain for simplicity. I'd vote for going with the `phantomOptions` for this work, then I'll add a FE Tech Talk topic around how we could support the HTML attribute approach in the future